src/components/Charts/LineChart.jsx

```diff
@@ -7,9 +7,11 @@ import { line, tooltip } from 'britecharts'
 import { connect } from 'react-redux'
 import { hashObject } from '../../utils'
 import { isDateEqual } from '../../utils/formatDate'
+import moment from 'moment';
```

Comment:
It makes sense to use moment since it's used in the app already, but since moment is deprecated we should transition at some point to dayjs. If you see anything that makes that transition easier while working with moment that'll probably help down the road.